tests/integrate/934_NO_Si2_tzdp_ns2_GPU/INPUT

```diff
@@ -0,0 +1,28 @@
+INPUT_PARAMETERS 
+#Parameters (1.General)
+suffix           autotest
+calculation      scf
+                 
+#nbands			 8
+symmetry         1
+                 
+#Parameters (2.Iteration)
+ecutwfc          50
+scf_thr          1e-5
```

Comment:
scf_thr is too large, could you change it to 1e-7 ?

Reply:
The scf_thr in all GPU test cases have now been set to 1e-7, but this will increase the testing time somewhat.